tests/benchmarks/test_parquet.py

```diff
@@ -26,12 +26,12 @@ def parquet_cluster():
 
 
 @pytest.fixture
-def parquet_client(parquet_cluster, sample_memory):
+def parquet_client(parquet_cluster, sample_memory, benchmark_time):
     with distributed.Client(parquet_cluster) as client:
         parquet_cluster.scale(N_WORKERS)
         client.wait_for_workers(N_WORKERS)
         client.restart()
```

Comment:
Yeah, this is a great example of things we don't want to include in the benchmark measurement!